docs/dev/proj.md

```diff
@@ -171,6 +171,10 @@ You can set the scope of the variables and secrets to **Repository** or **Organi
 
 ::::
 
+```{note}
```

Comment:
Let us remove this part since `RENOVATE_GIT_AUTHOR` is never mentioned above. The explanation may bring more confusion.